lib/generate-code.js

```diff
@@ -28,10 +28,20 @@ module.exports = function generateCode(
 
   var tempOutput = node.toString()
 
+  console.log(tempOutput)
+
+  if (
+    tempOutput.match(/\}\(.*.?Vue\);/) &&
+    tempOutput.includes('vue-class-component')
+  ) {
+    node.add(';exports.default = {...exports.default.__vccBase};')
```

Comment:
I think we will now end up with 2x `node.add(';exports.default = {...exports.default};')` (added here *and* line 43).

Can you 1) check if that is the case and 2) make sure we only add it once? If we aren't careful, this can become really complex really fast!

Reply:
node now ends with
```
';exports.default = {...exports.default.__vccBase};',
';exports.default = {...exports.default, render};' 
```
or 

```
';exports.default = {...exports.default.__vccBase};',
';exports.default = {...exports.default};' 
```

I think this is the desired behavoir because otherwise the code will be more complex.

Example:
```
if (
  tempOutput.match(/\}\(.*.?Vue\);/) &&
  tempOutput.includes('vue-class-component')
) {
if (tempOutput.includes('exports.render = render;')) {
    node.add(';exports.default = {...exports.default.__vccBase, render};')
  } else {
    node.add(';exports.default = {...exports.default.__vccBase};')
  }
} else {
  if (tempOutput.includes('exports.render = render;')) {
    node.add(';exports.default = {...exports.default, render};')
  } else {
    node.add(';exports.default = {...exports.default};')
  }
}
```
this will have an output of 
```
';exports.default = {...exports.default.__vccBase, render};'
```
or 

```
';exports.default = {...exports.default.__vccBase};'
```

Reply:
Right, true, that is very complex. Fair enough.

I wonder if we should create a `isClassComponent` function 🤔 

Anyway, this will do for now, I will test it out a bit and if it's all good do a release. thanks!